README.md

```diff
@@ -1,42 +1,39 @@
 # SCERES database
 
+
+##Conda environment
+
+Create a conda environment from the .yml file in the repository.
+```
+conda env create -f environment.yml
+```
+
 ## How to create the SCERES database ?
 
-1- Download the .sh and .sql scripts in your working directory, where you want the database to be.
+1- Clone the repository in your working directory, where you want the database to be.
```

Comment:
Instead of 
```
1- Clone the repository in your working directory, where you want the database to be.
```
I propose
```
1- Clone the repository in your working directory:

git clone https://github.com/Thibault-Poinsignon/sceres_database.git
```

You can also remove step 2.